Error trying to get Tuple of tensor with NewIValue

Hi, i'm trying to convert 2 tensors into a tuple. However when i try to use `ts.NewIValue([]ts.IValue{tensora, tensorb})` i'm getting this error: `NewIValue method call - 'slice -> struct' case - Unsupported type (slice)`. I managed to make it work by editing `tensor/jit.go` file in the line 90 with:
```
switch reflect.TypeOf(v).Elem().String() {
case "tensor.IValue":
```

Is this something that should be fixed? or i'm using `NewIValue` incorrectly?

```
panic: interface conversion: interface {} is []tensor.IValue, not []tensor.Tensor

goroutine 1 [running]:
github.com/sugarme/gotch/tensor.(*IValue).ToCIValue(0xc000050e70, 0x6314c0, 0xc000050df8, 0x6314c0)
	/home/truora/go/pkg/mod/github.com/sugarme/gotch@v0.3.9/tensor/jit.go:337 +0x2ee5
github.com/sugarme/gotch/tensor.(*CModule).ForwardIs(0xc0000b2040, 0xc000105f48, 0x1, 0x1, 0x0, 0x0, 0x0)
	/home/truora/go/pkg/mod/github.com/sugarme/gotch@v0.3.9/tensor/jit.go:1070 +0x3a5
main.main()
	/home/truora/go/src/bitbucket.org/truora/scrap-services/main.go:22 +0x265
exit status 2
```

--- a/tensor/jit.go
+++ b/tensor/jit.go
@@ -87,6 +87,7 @@ func NewIValue(v interface{}) *IValue {
 		retVal.kind = StringVal
 		retVal.name = "String"
 	case "slice":
+		fmt.Printf("slice elem type: %q\n", reflect.TypeOf(v).Elem().Kind().String())
 		switch reflect.TypeOf(v).Elem().Kind().String() {
 		case "IValue":
 			switch len(v.([]IValue)) {
@@ -124,20 +125,14 @@ func NewIValue(v interface{}) *IValue {
 			// 1. Tuple (Tensor, Tensor)
 			case val.Type() == reflect.TypeOf([]Tensor{}) && val.Len() == 2:
 				retVal.kind = TensorListVal
-				retVal.name = "TensorList"
-
-				// convert tensors to []ivalue
-				tensors := v.([]Tensor)
-				var ivals []IValue
-				for _, tensor := range tensors {
-					ival := NewIValue(tensor)
-					ivals = append(ivals, *ival)
-				}
-				retVal.value = ivals
+				retVal.name = "Tuple"
+				retVal.value = v.([]Tensor)
 
 				// 2. List (Tensor, Tensor, ...)
 			case val.Type() == reflect.TypeOf([]Tensor{}) && val.Len() > 2:
-				panic("Its Tensor list")
+				retVal.kind = TensorListVal
+				retVal.name = "TensorList"
+				retVal.value = v.([]Tensor)
 			default:
 				log.Fatalf("NewIValue method call - 'slice -> struct' case - Unsupported type (%v)\n", reflect.TypeOf(v).Kind().String())
 			}

